src/components/inputs/multi-select/ohri-multi-select.component.tsx

```diff
@@ -114,9 +119,9 @@ export const OHRIMultiSelect: React.FC<OHRIFormFieldProps> = ({ question, onChan
             readOnly={question.readonly}
           />
         </div>
-        <div className={styles.formField} style={{ marginTop: '0.125rem' }}>
+        <div style={{ marginTop: '0.125rem' }}>
```

Comment:
Is this a style that is used in other controls? Let's extract this to the scss file